tox.ini

```diff
@@ -22,3 +22,11 @@ basepython = python3
 deps = -r{toxinidir}/docs/requirements.txt
 skip_install = true
 commands = sphinx-build {posargs:-E} -b html docs dist/docs
+
+[testenv:man]
+whitelist_externals = make
+basepython = python3
+# deps = -r{toxinidir}/docs/requirements.txt
```

Comment:
same here, I believe it was done for testing purposes :)

Reply:
Yep. Actually I thought it could make sense to add the doc and manpage dependencies in a separate file. That makes it easier to append anything and we don't need to touch the `tox.ini` file. On the other hand, currently it's just sphinx.

There is still a little inconsistency between the targets "man" and "docs". The "man" target uses `make -C docs man` to build the manpage whereas the "docs" target uses `sphinx-build`.

I'm a big fan of consistency, so IMHO this should be consolidated. It would be better if both use the make call. Not sure if I should change that here as well... :thinking:

Reply:
I've corrected your suggestion in commit 30a5264